### PR TITLE
TDT blog: adding leaderboard screenshot

### PR DESCRIPTION
**Pull request by hainan-xv**
# What does this PR do ?

TDT blog: adding leaderboard screenshot
**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Jenkins CI
To run Jenkins, a NeMo User with write access must comment `jenkins` on the PR.

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [x] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: docs/blogs/posts/2024/2024-01-parakeet-tdt.md [hunk 1]
```diff
@@ -24,7 +24,12 @@ Earlier this month, we announced [Parakeet](https://huggingface.co/collections/n
...
+To put things in perspective, our Parakeet-TDT model with 1.1 billion parameters outperforms similar-sized Parakeet-RNNT-1.1b in accuracy, as measured as the average performance among 9 benchmarks on the [HuggingFace Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard). Notably, Parakeet-TDT is the first model to achieve an average WER below 7.0 on the leaderboard. Additionally, it achieves an impressive real-time factor (RTF) of 8.8e-3, 64% faster than Parakeet-RNNT-1.1b's RTF of 14.4e-3. Remarkably, Parakeet-TDT's RTF is even 40% faster than Parakeet-RNNT-0.6b (RTF 12.3), despite the latter having about half the model size.
+
+<figure markdown>
+  ![HuggingFace Leaderboard](leaderboard_01312024.png)
```
Comment (titu1994) [inline docs/blogs/posts/2024/2024-01-parakeet-tdt.md:30]: Don't embed image into the commit, follow the instruction in the readme to upload the image into NeMo release

**Review by nithinraok — APPROVED** [review]
LGTM

**Review by titu1994 — APPROVED** [review]

**Comment by titu1994** [comment]
Wait the URL is wrong

**Review by titu1994 — APPROVED** [review]
Thanks !

**Pull request merged**